docs/drive-management.md

```diff
@@ -0,0 +1,111 @@
+│ ID                  │ NODE   │ DRIVE │ SIZE    │ FILESYSTEM │ MAKE │ AVAILABLE │ DESCRIPTION │
+├─────────────────────┼────────┼───────┼─────────┼────────────┼──────┼───────────┼─────────────┤
+│ 252:16$ud8mwCjPT... │ master │ vdb   │ 512 MiB │ -          │ -    │ YES       │ -           │
+│ 252:16$gGz4UIuBj... │ node1  │ vdb   │ 512 MiB │ -          │ -    │ YES       │ -           │
+└─────────────────────┴────────┴───────┴─────────┴────────────┴──────┴───────────┴─────────────┘
+
+Generated 'drives.yaml' successfully.
+
+# Show generated drives.yaml file.
+$ cat drives.yaml
+version: v1
+nodes:
+    - name: master
+      drives:
+        - id: 252:16$ud8mwCjPTH8147TysmiQ2GGLpffqUht6bz7NtHqReJo=
+          name: vdb
+          size: 536870912
+          make: ""
+          select: "yes"
+    - name: node1
+      drives:
+        - id: 252:16$gGz4UIuBjQlO1KibOv7bZ+kEDk3UCeBneN/UJdqdQl4=
+          name: vdb
+          size: 536870912
+          make: ""
+          select: "yes"
+
+```
+
+2. Run `init` command.
+
+   The `init` command creates a request to add the selected drives in the YAML file generated using the `discover` command. As this process wipes out all data on the selected drives, wrong drive selection will lead to permanent data loss. Below is an example of `init` command:
+
+```sh
+$ kubectl directpv init drives.yaml
+
+ ███████████████████████████████████████████████████████████████████████████ 100%
+
+ Processed initialization request 'c24e22f5-d582-49ba-a883-2ce56909904e' for node 'master' ✔
+ Processed initialization request '7e38a453-88ed-412c-b146-03eef37b23bf' for node 'node1' ✔
+
+┌──────────────────────────────────────┬────────┬───────┬─────────┐
+│ REQUEST_ID                           │ NODE   │ DRIVE │ MESSAGE │
+├──────────────────────────────────────┼────────┼───────┼─────────┤
+│ c24e22f5-d582-49ba-a883-2ce56909904e │ master │ vdb   │ Success │
+│ 7e38a453-88ed-412c-b146-03eef37b23bf │ node1  │ vdb   │ Success │
+└──────────────────────────────────────┴────────┴───────┴─────────┘
+```
+
+Refer to the [discover command](./command-reference.md#discover-command) and the [init command](./command-reference.md#init-command) for more information.
+
+## List drives
+To get information of drives from DirectPV, run the `list drives` command. Below is an example:
+
+```sh
+$ kubectl directpv list drives
+┌────────┬──────┬──────┬─────────┬─────────┬─────────┬────────┐
+│ NODE   │ NAME │ MAKE │ SIZE    │ FREE    │ VOLUMES │ STATUS │
+├────────┼──────┼──────┼─────────┼─────────┼─────────┼────────┤
+│ master │ vdb  │ -    │ 512 MiB │ 506 MiB │ -       │ Ready  │
+│ node1  │ vdb  │ -    │ 512 MiB │ 506 MiB │ -       │ Ready  │
+└────────┴──────┴──────┴─────────┴─────────┴─────────┴────────┘
+```
+
+Refer to the [list drives command](./command-reference.md#drives-command) for more information.
+
+## Label drives
+Drives are labeled to set custom tagging which can be used in volume provisioning. Below is an example:
```

Comment:
can we also specify how to control scheduling using labels here? something like https://github.com/minio/directpv/blob/master/docs/scheduling.md#volume-scheduling-based-on-drive-labels

Reply:
It is covered in `volume-scheduling.md#Customizing drive selection`